_posts/news/2019-12-13-announcing-scalajs-1.0.0-RC2.md

```diff
@@ -0,0 +1,562 @@
+---
+layout: post
+title: Announcing Scala.js 1.0.0-RC2
+category: news
+tags: [releases]
+permalink: /news/2019/12/13/announcing-scalajs-1.0.0-RC2/
+---
+
+
+We are thrilled to announce the release of Scala.js 1.0.0-RC2!
+
+This release candidate is intended for testing purposes by as many users as possible, and as a synchronization point with library authors so that they can upgrade in preparation for the final release.
+If no critical issue is found until the end of January 2020, this RC will become the final release.
+
+**We encourage all users who are able to do so to test their projects with this RC, and report any issue as soon as possible.**
+
+As the change in major version number witnesses, this release is *not* binary compatible with 0.6.x, nor with the previous milestones and RCs of the 1.x series.
+Libraries need to be recompiled and republished using this RC to be compatible.
+Moreover, this release is not entirely source compatible with 0.6.x either.
+
+These release notes contain cumulative changes with respect to 0.6.31.
+Compared to 1.0.0-RC1, this release candidate contains the following changes:
+
+* Fix the handling of versions of Scala.js to correctly work post-v1.0.0
```

Comment:
work correctly?

Reply:
Hum I'm bad with placing adverbs in English, but I thought they were supposed to come before the verb, no?

Reply:
["to work correctly"][1]: 31e6 Google results
["to correctly work"][2]: 87e3 Google results

:P

[1]: https://www.google.com/search?sxsrf=ACYBGNTLB7QEccmmSeuhFQQccPNl3lDNZA%3A1576171540357&ei=FHjyXZqxFYe9UNexqrAK&q=%22to+work+correctly%22&oq=%22to+work+correctly%22&gs_l=psy-ab.3..0i7i30l4j0i30.9248.12505..12823...0.2..0.279.1002.2-4......0....1..gws-wiz.......0i71j0i5i30.gUZqAU6U4Zs&ved=0ahUKEwiai5LG0LDmAhWHHhQKHdeYCqYQ4dUDCAs&uact=5
[2]: https://www.google.com/search?sxsrf=ACYBGNTLB7QEccmmSeuhFQQccPNl3lDNZA%3A1576171540357&ei=FHjyXZqxFYe9UNexqrAK&q=%22to+work+correctly%22&oq=%22to+work+correctly%22&gs_l=psy-ab.3..0i7i30l4j0i30.9248.12505..12823...0.2..0.279.1002.2-4......0....1..gws-wiz.......0i71j0i5i30.gUZqAU6U4Zs&ved=0ahUKEwiai5LG0LDmAhWHHhQKHdeYCqYQ4dUDCAs&uact=5